core/css.js

```diff
@@ -1067,8 +1069,6 @@ Blockly.Css.CONTENT = [
     'outline: none;',
     'padding: 4px 0;',
     'position: absolute;',
-    'overflow-y: auto;',
```

Comment:
Hey @shakao, just wondering why this was removed?

Reply:
it was causing an unnecessary scrollbar to appear in our context menu--i think actually the better fix would be to put this back and add content-box sizing to the .goog-menu; i'll do that in the next batch of updates